test_output/help/server_delete.txt

```diff
@@ -3,6 +3,7 @@ knife bmcs server delete --help
 knife bmcs server delete (options)
         --bmcs-config-file FILE      The path to the Oracle BMCS config file. Default: ~/.oraclebmc/config
         --bmcs-profile PROFILE       The profile to load from the Oracle BMCS config file. Default: DEFAULT
+    -N, --node-name NAME             The Chef node name being removed. If not specified, the instance display name will be used.
```

Comment:
The name of the Chef node to be removed when using the purge option. If not specified, the instance display name will be used.